.pipelines/soak-tests-template.yml

```diff
@@ -63,9 +63,7 @@ jobs:
             kubectl cluster-info
           displayName: "Check cluster's health"
 
-        - script: |
-            export REGISTRY="${REGISTRY:-$(REGISTRY_NAME).azurecr.io/k8s/aad-pod-identity}"
-            make e2e
+        - script: make e2e
```

Comment:
Letting soak tests use the default registry defined in `test/e2e/framework/config.go`